cmd/system-probe/config/config.go

```diff
@@ -154,6 +158,9 @@ func load() (*Config, error) {
 	if cfg.GetBool(diNS("enabled")) {
 		c.EnabledModules[DynamicInstrumentationModule] = struct{}{}
 	}
+	if cfg.GetBool(wcdNS("enabled")) {
+		c.EnabledModules[WindowsCrashDetectModule] = struct{}{}
```

Comment:
Why are there two enable config keys?

Reply:
good catch.  We moved the config around.  that should be removed.